heidrun/ia.rb

```diff
@@ -0,0 +1,238 @@
+# coding: utf-8
```

Comment:
Is it necessary to specify this if we're using Ruby 2, which defaults to UTF8 (as I understand it)?

Obviously a minor issue, but it just caught my eye and made me curious.